flaskbb/user/views.py

```diff
@@ -141,18 +146,42 @@ def get(self, username):
         return render_template("user/profile.html", user=user)
 
 
-register_view(user, routes=['/settings/email'], view_func=ChangeEmail.as_view('change_email'))
-register_view(user, routes=['/settings/general'], view_func=UserSettings.as_view('settings'))
-register_view(
-    user, routes=['/settings/password'], view_func=ChangePassword.as_view('change_password')
-)
-register_view(
-    user,
-    routes=["/settings/user-details"],
-    view_func=ChangeUserDetails.as_view('change_user_details')
-)
-register_view(user, routes=['/<username>/posts'], view_func=AllUserPosts.as_view('view_all_posts'))
-register_view(
-    user, routes=['/<username>/topics'], view_func=AllUserTopics.as_view('view_all_topics')
-)
-register_view(user, routes=['/<username>'], view_func=UserProfile.as_view('profile'))
+@impl
```

Comment:
We probably want to use ``tryfirst`` here (even though it [doesn't guarantee](http://pluggy.readthedocs.io/en/latest/#call-time-order) that it will be called first when another hook is marked as ``tryfirst``)

Reply:
Do we care what order blueprints are registered in though?

Reply:
I think that we should prefer the internal blueprints over a plugins blueprint. It might confuse the user a little bit when flaskbb's blueprint is conflicting with his instead of the other way around. In the end it probably doesn't really matter though.

Reply:
That's a fair point. I'll tag internal blueprints.